pallets/detach/src/lib.rs

```diff
@@ -226,19 +244,23 @@ pub mod pallet {
 	#[pallet::generate_store(pub(super) trait Store)]
 	pub struct Pallet<T>(_);
 
+	/// StorageValue that represents a list of `DetachRequests`
 	#[pallet::storage]
 	pub type DetachRequests<T: Config> = StorageValue<_, Vec<DetachRequest>, ValueQuery>;
 
 	#[pallet::storage]
 	pub type DetachNonces<T: Config> =
 		StorageDoubleMap<_, Blake2_128Concat, Vec<u8>, Blake2_128Concat, SupportedChains, u64>;
 
+
 	#[pallet::storage]
 	pub type DetachedHashes<T: Config> = StorageMap<_, Identity, Hash256, ExportData>;
 
+	/// EthereumAuthorities is a StorageValue that keeps track of the set of ECDSA public keys of the Ethereum accounts on which a Proto-Fragment can be detached from this Blockchain
 	#[pallet::storage]
 	pub type EthereumAuthorities<T: Config> = StorageValue<_, BTreeSet<ecdsa::Public>, ValueQuery>;
 
+	/// ¿
```

Comment:
These are the public keys representing the actual keys that can Sign messages to present to external chains to detach onto